### DISCUSSION TITLE
Local host commands (?)

### DISCUSSION THREAD

**Discussion by Rom0067** [Pryaxis/TShock#3030] [Q&A]
I'm new to self hosting and docker game servers so I apologize in advance...

I managed to get a terraria server started on my Linux server with docker.  Other then not having a setup code ( I see there is a walkthrough here I'm going to try soon.) I was wondering if there is a way to run commands from my Linux server.  I was able to do it when I restarted the server with a previously made world.  Is there a way to pull it up? Sorry if this is a dumb question please be gentle lol

**Comment by hakusaro** [comment]
It's not a dumb question, but, which image are you using? Most likely you're better off just running the commands ingame.

**Reply by Rom0067** [reply]
Thanks for your kindness.  I am using   ryshe/terraria:latest.  It showed me more commands options in the help menu that's why I was thinking it might be better for a noobie

**Comment by sgkoishi** [comment]
There's no setup code once you register. Try [docker container attach](https://docs.docker.com/reference/cli/docker/container/attach/) if you want to interact with it?

**Reply by Rom0067** [reply]
I tried attach and my cli just hung up nothing happened.  Thank you though.

**Comment by Rom0067** [comment]
So I did a little more exploring and most likely not surprising to most of you but I made a discovery lol. The attached picture is what I am looking to use (in case I didn't explain it right). I got there by stopping container and running -it in place of -d. Again no surprise I'm just happy with myself. Anyway using chatgpt to bounce ideas off it seems like the initial run commands doesnt have stdin so because of that I can't attach to a running container unless stop/restart with -it.  I was hoping to be able to login while server is up to issue a command real quick without opening the game (kids playing Im busy etc)

![Screenshot_20240510_080525_Termius](https://github.com/Pryaxis/TShock/assets/71397859/d50bb593-8607-46ab-ae98-2e8647023e74)
